app/assets/javascripts/angular/controller/static_pages/navbar.js.erb

```diff
@@ -15,4 +15,18 @@ app.controller('NavbarCtrl', ['$scope', '$auth', '$location','$window', function
       console.log(response);
     });
   };
+
+  $scope.title = ''
+  switch($location.url().replace(/forms/,'').replace(/\d/,'')) {
```

Comment:
Are you trying to display the right title in Navbar, but they have different controllers, hence different $scope?

The correct way to communicate across controllers is this http://stackoverflow.com/questions/11252780/whats-the-correct-way-to-communicate-between-controllers-in-angularjs